risc0/zkvm/src/receipt/composite.rs

```diff
@@ -221,3 +242,54 @@ impl CompositeReceipt {
         ))
     }
 }
+
+/// Verifier parameters for [CompositeReceipt][super::CompositeReceipt].
+///
+/// [CompositeReceipt][super::CompositeReceipt] is a collection of individual receipts that
+/// collectively  prove a claim. It can contain any of the individual receipt types, and so it's
+/// verifier is a combination of the verifiers for every other receipt type.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[non_exhaustive]
+pub struct CompositeReceiptVerifierParameters {
+    /// Verifier parameters related to [SegmentReceipt].
+    pub segment: SegmentReceiptVerifierParameters,
+    /// Verifier parameters related to [SuccinctReceipt].
+    pub succinct: SuccinctReceiptVerifierParameters,
+    /// Verifier parameters related to [CompactReceipt].
+    pub compact: CompactReceiptVerifierParameters,
+}
+
+impl Digestible for CompositeReceiptVerifierParameters {
```

Comment:
I'm kind of unsure why we'd need to worry about making the metadata digestible if it can't impact the claim or attestations.

Reply:
And to be clear, the `ReceiptMetadata` as a whole does no define a digest method, and it wouldn't make much sense to do so.

https://github.com/risc0/risc0/pull/1764/files/7e5c06f843847db1928b41b5e195eca45b738a47#diff-63cd47df26551e5cfb26448b6449ba92432b6f5a277788d6000168d5e0569189R412